### ISSUE TITLE
Prototype Hue table / Superset dashboard for categorical data

### ISSUE THREAD

**Issue by durandom** [operate-first/continuous-deployment#29]

**Comment by hemajv** [comment]
Creating a Hue table for the categorical dataset stored [#28] and importing a dashboard for the same dataset into superset
Categorical dataset: https://github.com/aicoe-aiops/categorical-encoding/tree/master/notebooks/data

**Comment by tumido** [comment]
@hemajv @durandom 
I've just noticed, Hue is not yet part of the ODH (the Kubeflow based). That's probably a big blocker for this task:
![image](https://user-images.githubusercontent.com/7453394/94254022-71b8d480-ff26-11ea-8784-8f29c4fcf944.png)


Ref: https://github.com/opendatahub-io/odh-manifests

**Comment by durandom** [comment]
🤦 
@rimolive any updates on when the data catalog would make it into ODH?

**Comment by durandom** [comment]
on hold until we have a data catalog

**Comment by rimolive** [comment]
We started the migration of Data Catalog but migrating each component at a time. The first component will be Thrift Server.

Even if we don't have Hue yet migrated to ODH you can use any SQL Client with Hive support to interact with Thrift Server.

We expect to finish the Thrift Server migration in our current sprint which will be in the next 3 weeks.

**Comment by tumido** [comment]
does that mean, once Thrift server is available can we create a Hive table that is consumable from Superset via that Thrift server's SQL client?

**Comment by rimolive** [comment]
@tumido Correct. Thrift Server is the entrypoint for core Data Catalog features. Any SQL Client can interact with Data Catalog that way.

**Comment by hemajv** [comment]
On that note, for starters I have been playing around with `beeline`: https://cwiki.apache.org/confluence/display/Hive/HiveServer2+Clients#HiveServer2Clients-Beeline%E2%80%93NewCommandLineShell to interact with a stage instance of Thriftserver (that @rimolive maintains) and managed to run a script that can create tables

**Comment by tumido** [comment]
@hemajv Beeline sounds great to me. It seems to be able to "apply" a `sql` file on Hive, which is basically all we need now. :+1:

**Comment by hemajv** [comment]
@tumido yes, that's exactly what I was able to do :)

**Comment by durandom** [comment]
@anishasthana what's the diff in the CRDs for the "old" ODH operator that supports DataCatalog? Could we easily go with the older version and switch to the new one later?

**Comment by durandom** [comment]
ok, nvm, doesnt seem to work, since we can't deploy 2 versions of the operator...
